main/acle.md

```diff
@@ -6515,7 +6714,7 @@ single vectors:
 
 | **Signed integer**   | **Unsigned integer** | **Floating-point**   |                      |
 | -------------------- | -------------------- | -------------------- | -------------------- |
-| `svint8_t`           | `svuint8_t`          |                      |                      |
+| `svint8_t`           | `svuint8_t`          |                      | `svmfloat8_t         |
```

Comment:
Should the 

> `svbfloat16_t` is only available if the header file also provides a definition of `bfloat16_t`  

section below be repeated or extended for svmfloat8_t?

Reply:
Wouldn't that be necessary only if the underlying type definition was conditional?

Reply:
It would, I was confused, thanks for clarifying :)